src/main/java/training/subsystems/Elbow/Elbow.java

```diff
@@ -0,0 +1,83 @@
+package training.subsystems.Elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.subsystems.Elbow.ElbowConstants.*;
+
+public class Elbow extends GBSubsystem {
+    private static Elbow instance;
+
+    public static Elbow getInstance() {
+        if (instance == null) {
+            instance = new Elbow();
+        }
+        return instance;
+    }
+
+    private final CANSparkMax motor;
+    private final SparkPIDController motorPIDF;
+    private Rotation2d targetAngle;
+
+    private Elbow() {
+        targetAngle = DEFAULT_POSITION_ELBOW;
+
+        motor = new CANSparkMax(ELBOW_ID, CANSparkLowLevel.MotorType.kBrushless);
+        motorPIDF = motor.getPIDController();
+
+        motorPIDF.setP(ELBOW_PID_CONTROLLER.getP());
+        motorPIDF.setD(ELBOW_PID_CONTROLLER.getD());
+        motorPIDF.setI(ELBOW_PID_CONTROLLER.getI());
+        motorPIDF.setOutputRange(0, POWER_LIMIT_ELBOW);
```

Comment:
r u sure?, i think it should be 
```suggestion
        motorPIDF.setOutputRange(-POWER_LIMIT_ELBOW, POWER_LIMIT_ELBOW);
```